host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_nvcc_c++17.cmake

```diff
@@ -45,7 +45,7 @@ set(MPI_CXX_COMPILER "${_MPI_BASE_DIR}/bin/mpicxx" CACHE PATH "")
 #------------------------------------------------------------------------------
 # Cuda
 #------------------------------------------------------------------------------
-
+# _blt_tutorial_cuda_config_start
```

Comment:
Ah you got halfway there... there is a file path in the documentation that goes along with this. If i remember correctly it also names a machine that should be updated. (Surface -> Lassen, i think)